download/index.md

```diff
@@ -1,6 +1,7 @@
 ---
-layout: downloadpage
+layout: download
 title: Download
+<<<<<<< HEAD
```

Comment:
You need to fix this conflict.

Reply:
Sorry -- I fixed it, but forgot to save before adding... My bad.